c/driver_manager/adbc_driver_manager.cc

```diff
@@ -816,6 +846,13 @@ AdbcStatusCode AdbcLoadDriverFromInitFunc(AdbcDriverInitFunc init_func, int vers
     FILL_DEFAULT(driver, StatementSetSqlQuery);
     FILL_DEFAULT(driver, StatementSetSubstraitPlan);
   }
+  if (version >= ADBC_VERSION_1_1_0) {
+    auto* driver = reinterpret_cast<struct AdbcDriver*>(raw_driver);
+    FILL_DEFAULT(driver, StatementExecuteSchema);
```

Comment:
shouldn't this also have the same for the other new functions? `AdbcStatementCancel` and `AdbcStatementGetDouble`?

Reply:
I'm going to defer implementation, just wanted to demonstrate the basic idea worked. I just want to get the API sort of in the right shape.

Reply:
Agh, let me retarget this PR against the branch, not main.

Reply:
There - the intent is to build this out on a branch then manually ff-merge it at the end. If the approach looks reasonable I'm also going to build out more serious compatibility tests.

Reply:
Gotcha, okay then in that case I'm in favor of this approach.